### ISSUE TITLE
CNN development

### ISSUE THREAD

**Issue by Mmiglio** [Mmiglio/SpeechRecognition#5]
Test different training method (e.g. optimisers, lr schedulers, ... ) for the CNN model. 

Useful resources:

* [Convolutional Neural Networks for Small-footprint Keyword Spotting](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/43969.pdf)

**Comment by Mmiglio** [comment]
With default model architecture, Adam optimizer  and reducing learning rate on plateau we obtain good results:

* Accuracy: 0.9611
* Precision: 0.9617
* Recall: 0.9636
* F1 score: 0.9625
